demo/settings.cfg

```diff
@@ -2,3 +2,9 @@
 ESHost = http://localhost:9200
 //ESUser = ltr_demo
 //ESPassword = changeme
+RanklibJar = RankLib-2.8.jar
```

Comment:
We might want a version of RankyMcRankFace, as it has output of feature importance